src/pfDataSource.Common/CsvBuilder.cs

```diff
@@ -75,6 +75,11 @@ public Task<bool> WriteAsync(string path)
 				return Task.FromResult(false);
 			}
 		}
+
```

Comment:
Is there a reason we have our own CSV Builder class rather then using a standard CSV package like https://joshclose.github.io/CsvHelper/

Reply:
ask @daniel-buchanan ;-)

Reply:
Well update the csv build in another PR